tox/session.py

```diff
@@ -34,10 +34,12 @@ def prepare(args):
     return config
 
 
-def main(args=None):
+def main(args):
```

Comment:
This change breaks the `tox` startup executable set up by setuptools. When that startup executable runs `tox`'s `main()` function (and this is the one used), it does not pass it any arguments.

Reply:
@gaborbernat - And we might want to add an external process test making sure those setuptools installed starter executables work correctly in the future?